### PR TITLE
Add mustache message rendering

### PR DESCRIPTION
**Pull request by maxjustus**
## To test
- start rails server and sidekiq
- start mailcatcher `bundle exec mailcatcher`
- go to locahost:1080
- create some subscribers
- post to the messages endpoint with a message that has some mustache in it `Hi {{name}} this is your email: {{email}}`
- check mailcatcher for your emails
- validate they're validity

### CODE REVIEW

File: app/services/render_message.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
+class RenderMessage
```
Comment (hexgnu) [inline app/services/render_message.rb:3]: Needs a top level comment documenting what the purpose of this file is.

File: lib/mustache_template_cache.rb [hunk 2]
```diff
@@ -0,0 +1,15 @@
+module MustacheTemplateCache
+  extend self
```
Comment (hexgnu) [inline lib/mustache_template_cache.rb:2]: `module_function :fetch, :cache, :clear` is a more explicit way to do this

File: spec/services/render_message_spec.rb [hunk 3]
```diff
@@ -0,0 +1,11 @@
+require 'spec_helper'
+
+describe RenderMessage do
+  it 'renders passed template with passed subscriber' do
+    sub = Fabricate(:subscriber)
+    template = Fabricate(:message, html: '{{name}}, {{email}}')
+    out = described_class.new(template, sub).html
+    expect(out).to include(sub.email)
```
Comment (hexgnu) [inline spec/services/render_message_spec.rb]: I almost feel like this should just be `expect(out).to eq ("#{name}, #{email}")` but this works too :smile:

**Comment by hexgnu** [comment]
super awesome :sparkles: clean

File: spec/services/render_message_spec.rb (hunk 3 above)
Comment (maxjustus) [inline spec/services/render_message_spec.rb]: oh, duh

File: lib/mustache_template_cache.rb (hunk 2 above)
Comment (maxjustus) [inline lib/mustache_template_cache.rb:2]: oh kewl

**Comment by cmhobbs** [comment]
Let's put your commit info into the README until we get a Procfile with some foreman action.

File: app/services/render_message.rb (hunk 1 above)
Comment (cmhobbs) [inline app/services/render_message.rb:3]: +1

**Comment by cmhobbs** [comment]
:cake: once docs are done

**Comment by maxjustus** [comment]
Merge me I'm Irish!

**Pull request merged**